docs/03-architecture/07-substrate.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: /architecture/substrate
+id: architecture-substrate
+title: Substrate Support
+description: The following details Substrate support on Sygma.
+---
+
+:::info
+The following section details Sygma's compatibility with Polkadot and Kusama parachains via its Substrate pallet support. 
+:::
+
+### Substrate Support
+
+A unique feature of the Sygma protocol is its ability to provide interoperability between EVM-based and Substrate-based blockchains. This includes those [parachains](https://wiki.polkadot.network/docs/learn-parachains) deployed under Polkadot and Kusama's [shared security model](https://wiki.polkadot.network/docs/learn-architecture).
+
+Similar to how Sygma enables EVM support by invoking various smart contracts, Sygma provides Substrate support via interactions with [custom Sygma pallets](https://github.com/sygmaprotocol/sygma-substrate-pallets). 
+
+:::info
+Substrate pallets provide a modular approach to building Substrate-based blockchains and contain runtime logic which can be used, modified, and extended accordingly. 
+:::
+
+---
+
+### Key Concepts
+
+Sygma's custom pallets contains a multitude of different identifiers with complex interrelationships. These may appear overwhelming at first for EVM developers crossing the chasm, but we will explain the concepts below:
+
+- **MultiLocation**: `MultiLocation` is a cross-consensus message format (XCM) concept used to identify any single entity location that exists within the world of Polkadot consensus. `MultiLocation` always expresses a relative location to the current location. Practically, `MultiLocations` are used to identify places to send XCM messages. In Sygma pallets, it is used to identify the destination when depositing.
+- **MultiAsset**: `MultiAsset` is a Substrate type that identifies whether assets are fungible or non-fungible, native or foreign, etc. It is used to handle multiple assets in the Polkadot world. In Sygma, `MultiAsset` is used to identify the asset despite its location and fungibility. 
+-  **DomainID & ChainID**: In the context of the Sygma pallets, a `DomainID` is a unique identifier for a particular domain, and a `ChainID` is a unique identifier for a specific blockchain. These IDs are registered together, so each `DomainID` is associated with a specific `ChainID`. This information is stored in the chain's storage, and you can look up which `ChainID` is associated with a particular `DomainID`.
+-  **ResourceID**: This is a unique identifier for an asset in Sygma. It's linked with an XCM asset through a mapping defined in the runtime, which associates an `AssetId` with a `ResourceID`.
```

Comment:
Add: this mapping is known as `ResourcePairs`

Reply:
added